tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -938,38 +1007,21 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
         inp_model_is_object = input_model_is_object(args)
         if inp_model_is_object:
             model_framework = check_model_object(args)
-            if model_framework == "pytorch" and not os.environ.get('USE_PYTORCH_FRONTEND'):
-
-                opset_version = None
-                if 'onnx_opset_version' in args and args['onnx_opset_version'] is not None:
-                    opset_version = args['onnx_opset_version']
-
+            if model_framework == "pytorch":
+                use_pt_fe = not args.get("use_legacy_frontend", False)
```

Comment:
I think we should have a deal with the default value of the option here. It is already defined in the common place. Can we do as follows:

```
use_pt_fe = not args['use_legacy_frontend']
```